Light/src/rendering/renderer.cpp

```diff
@@ -60,6 +70,41 @@ namespace Light
 			}
 		}
 
+		for (int i = 0; i < 4; i++)
+		{
+			if (i < (int)s_sceneData->spotLights.size())
```

Comment:
Prefer C++ style casts (aka `static_cast`) over C style casts. Why is this cast necessary, anyways?

Reply:
I believe, the cast was present in the original code as well, so wrote my code following that style. Yeah, just checked, even without that it runs fine.